documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
+= Basic Maven Configuration
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+
+In the context of application development, Maven allows you to extend your applications by referencing and utilizing resources from local and remote repositories. This article serves as an introduction to using Maven with the IDE and contains instructions for the following tasks:
+
+. <<New_Project,Create a New Maven Project>>
```

Comment:
I didn't realize there was an automatic ToC available. At a quick look, I can't find anything beyond instructions to add :toc: to the definitions at the beginning of the document and that doesn't seem to quite do the trick by itself. I'll figure this out and add it for future docs. I used the recommended method in the z_howto template here since it appeared to be the recommended way.